server/index.js

```diff
@@ -17,6 +18,7 @@ const PORT = process.env.PORT || 3000;
 app.use('/documentation/v1/', swaggerUI.serve, swaggerUI.setup(docs));
 
 app.use('/api/v1/', routes);
```

Comment:
you are replacing v1 with v2 I don't think you need to keep v1

Reply:
Okay thanks, I didn't know that let me fix it!.